addons/website_forum/static/src/css/website_forum.sass

```diff
@@ -103,3 +107,34 @@ a.no-decoration
 
 button.btn-link.text-muted
   color: #999
+
+.user-tooltip
+  position: absolute
+  z-index: 200
+  display: none
+
+.user_heading
```

Comment:
We really need this class. Default margin and padding of html heading tags makes the tool tip look very ugly and bigger than necessary. Also some of the themes (for example cyborg) increases the font size of heading drastically that will also ruin the tool tip. So I gave it a uniform size.